params/precompile_config.go

```diff
@@ -66,7 +66,7 @@ func (p *PrecompileUpgrade) getByKey(key precompileKey) (precompile.StatefulPrec
 // - the specified blockTimestamps must be compatible with those
 //   specified in the chainConfig by genesis.
 // - check a precompile is disabled before it is re-enabled
-func (c *ChainConfig) VerifyPrecompileUpgrades() error {
+func (c *ChainConfig) verifyPrecompileUpgrades() error {
```

Comment:
nit: should update comment to `verifyPrecompileUpgrades`

Reply:
done